app/src/main/res/values/drawables.xml

```diff
@@ -0,0 +1,8 @@
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Where are these being used? If nowhere, please remove.

Reply:
These were used in navigation drawer menu items as per the mock , all items from menu other than help and home , as well as its corresponding images are removed now